packages/main/cypress/specs/DayPicker.cy.tsx

```diff
@@ -0,0 +1,79 @@
+import "../../src/Assets.js";
+import DayPicker from "../../src/DayPicker.js";
+import { getLanguage, setLanguage } from "@ui5/webcomponents-base/dist/config/Language.js";
+import "@ui5/webcomponents-localization/dist/features/calendar/Gregorian.js";
+
+function DefaultDayPicker() {
+	return <DayPicker id="daypicker"/>;
+}
+
+describe("Day Picker Tests", () => {
+	it("Select day with Space", () => {
+		cy.mount(<DefaultDayPicker />);
+		cy.get("#daypicker")
+			.shadow()
+			.find(".ui5-dp-item--now")
+			.as("today");
+		cy.get("@today")
+			.realClick()
+			.realPress("ArrowRight")
+			.realPress("Space");
+		cy.focused()
+			.invoke("attr", "data-sap-timestamp")
+			.then(timestampAttr => {
+				const timestamp = parseInt(timestampAttr!);
+				const selectedDate = new Date(timestamp * 1000).getDate();
+				const expectedDate = new Date(Date.now() + 24 * 3600 * 1000).getDate();
+				expect(selectedDate).to.eq(expectedDate);
+			});
+	});
+
+	it("Select day with Enter", () => {
+		cy.mount(<DefaultDayPicker />);
+		cy.get("#daypicker")
+			.shadow()
+			.find(".ui5-dp-item--now")
+			.as("today");
+
+		cy.get("@today")
+			.realClick()
+			.realPress("ArrowRight")
+			.realPress("Enter");
+
+		cy.focused()
+			.invoke("attr", "data-sap-timestamp")
+			.then(timestampAttr => {
+				const timestamp = parseInt(timestampAttr!);
+				const selectedDate = new Date(timestamp * 1000).getDate();
+				const expectedDate = new Date(Date.now() + 24 * 3600 * 1000).getDate();
+				expect(selectedDate).to.eq(expectedDate);
+			});
+	});
+
+	it("Day names are correctly displayed when length is less than 3", () => {
```

Comment:
Before mounting we should set the language again to "english" as changing the configuration affects all tests.  That is why this test fails.

Reply:
This test isn't failing even once on my end (re-triggered the tests more than 15 times). I think we should sync this offline.